### PR TITLE
Happy thoughts Api - Caroline Birgersson

### PR DESCRIPTION
**Pull request by carolinebirgersson**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -20,9 +38,41 @@ app.use(bodyParser.json())
...
+  res.send('This is the Happy Thoughts API by Caroline Birgersson')
+})
+
+app.get('/thoughts', async (req, res) => { //finds the added thoughts
+  try {
+    const thoughts = await Thought.find().sort({createdAt: 'desc'}).limit(20).exec()
+    res.status(201).json(thoughts)
+  } catch (err) {
+    res.status(400).json({message: 'could not load the database' })
+  }
```
Comment (johannarexin) [inline server.js:50]: Nice and clean, good :)

File: server.js [hunk 2]
```diff
@@ -3,15 +3,33 @@ import bodyParser from 'body-parser'
...
+  },
+  hearts: {
+    type: Number,
+    default: 0
+  },
+  createdAt: {
+    type: Date,
+    default: Date.now
+  }
+})
```
Comment (johannarexin) [inline server.js:30]: Nice model!

**Review by johannarexin — COMMENTED** [review]
Great job with this weeks project Caroline :) Your code is clean and easy to follow! Maybe you could add a link to your deployed Heroku page as well?

File: README.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+The mission this week was to use our new skills with Express and Mongodb to build an API that includes both GET request endpoints to return data and POST endpoints to create data. 
+I learned:
+- How to build a full API which includes handling of user input
+- How to include error handling to return good validation errors
+- How to build an API which works well with an existing frontend
 
 ## View it live
-
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://spread-happy-thoughts.netlify.app/
```
Comment (KarolCislowski) [inline README.md:13]: Please add an address to the hosted API.

**Review by KarolCislowski — APPROVED** [review]
Very nice application, but remember to add a link to the hosted app inside your read me 👍🏻

**Pull request closed without merging**